cmake/ecbuild_system.cmake

```
@@ -211,6 +211,7 @@ if( NOT ECBUILD_SYSTEM_INITIALISED )
     include( ecbuild_append_to_rpath )
     include( ecbuild_download_resource )
     include( ecbuild_get_test_data )
+    include( ecbuild_check_url )
```

```suggestion
    include( ecbuild_check_urls )
```
once the file has been renamed